asminc/ascii_charmap.inc

```diff
@@ -0,0 +1,291 @@
+;/*****************************************************************************/
+;/*                                                                           */
+;/*                              ascii_charmap.inc                            */
+;/*                                                                           */
+;/*   No translations, encodings are stored as they were typed in the host.   */
+;/*                                                                           */
+;/*                                                                           */
+;/* 2019-09-07, Greg King                                                     */
+;/*                                                                           */
+;/* This software is provided "as-is", without any expressed or implied       */
+;/* warranty.  In no event will the authors be held liable for any damages    */
+;/* arising from the use of this software.                                    */
+;/*                                                                           */
+;/* Permission is granted to anyone to use this software for any purpose,     */
+;/* including commercial applications, and to alter it and redistribute it    */
+;/* freely, subject to the following restrictions:                            */
+;/*                                                                           */
+;/* 1. The origin of this software must not be misrepresented; you must not   */
+;/*    claim that you wrote the original software. If you use this software   */
+;/*    in a product, an acknowledgment in the product documentation would be  */
+;/*    appreciated, but is not required.                                      */
+;/* 2. Altered source versions must be plainly marked as such, and must not   */
+;/*    be misrepresented as being the original software.                      */
+;/* 3. This notice must not be removed or altered from any source             */
+;/*    distribution.                                                          */
+;/*                                                                           */
+;/*****************************************************************************/
+
+;/* No include guard here.  Each charmap header
+;** may be included many times in a source file.
+;*/
```

Comment:
As far as I know there's nothing like include guards for asm .inc files. From that perspective the comment asks to be removed completely, doesn't it?

Reply:
I can have:
```
.ifndef ASCII_CHARMAP
.define ASCII_CHARMAP 1
.endif
```
But I will remove if you want, not a problem. I just converted the original files automatically.

Reply:
> But I will remove if you want, not a problem.

Please do so, thanks in advance!

Reply:
> As far as I know there's nothing like include guards for asm .inc files. From that perspective the comment asks to be removed completely, doesn't it?

Sorry, I clicked something and it resolved this issue, not my intention.